### ISSUE TITLE
[bug 9c63e7] SSA: set the default color to white & swap red/blue channels

### ISSUE THREAD

**Issue by Gorrrg** [PetterKraabol/Twitch-Chat-Downloader#7]
```
if color is None:
                color = '000000'
```

Gives me black text with a black border in MPC-HC. Should be ```color = 'FFFFFF'``` -> white text with black border.

Also RED/BLUE must be byteswapped, i.e. reverse the order of the three bytes: RGB -> BGR (ssa uses BGR it seems).

**Comment by Gorrrg** [comment]
I fixed the RGB/BGR issue for myself. First I copypasted a function I found [on the web](http://stackoverflow.com/questions/5864271/reverse-a-string-in-python-two-characters-at-a-time-network-byte-order) that solved this exact problem, and seemed to be decent without any type conversion:

```
def rev(a):
    new = ""

    for x in range(-1, -len(a), -2):
        new += a[x-1] + a[x]

    return new
```

And then of course I replaced the `000000` with `FFFFFF` for the null color. Haven't tested it extensively but by looking at the SSA file it seems to be ok.
